### ISSUE TITLE
Garages names?

### ISSUE THREAD

**Issue by iSentrie** [lilfuchs/esx_drp_garage#37]
![asd](https://user-images.githubusercontent.com/29688478/61802552-52fa1200-ae39-11e9-8bf8-af3b2912beca.jpg)

Possible to make a names for each garage?

**Comment by OhTanoshi** [comment]
Yes its possibly i sadly dont have away to do it and test it(i also thought it was already in but its been forever). it should be really simple if you look at other resources and such you should be able to do it easily

**Comment by KaMiKaZeE1** [comment]
Incase you didn't work it out garage names are controlled by the name set in the config.lua

Example - Garage_Poissoniers

Just change the it to whatever you want after "Garage_"

Example - Garage_Fishmonger
